packages/opencensus-instrumentation-http2/src/http2.ts

```diff
@@ -178,7 +178,10 @@ export class Http2Plugin extends HttpPlugin {
         const userAgent =
           headers['user-agent'] || headers['User-Agent'] || null;
 
-        const host = url.parse(authority).host;
+        const host = (authority instanceof url.URL
```

Comment:
Thanks @StoneDot for the fix! Would you be willing to add a unit test for this?

Reply:
+1, also update the CHANGELOG.md

Reply:
I've written the tests for this PR, but it hasn't worked correctly because of another bug (See https://github.com/census-instrumentation/opencensus-node/issues/641).

Could I tweak the tests to pass?

Reply:
Also I've updated CHANGELOG.md at #de29571